scripts/mergehex.py

```diff
@@ -41,14 +41,17 @@ def parse_args():
     parser.add_argument("-o", "--output", required=False, default="merged.hex",
                         type=argparse.FileType('w', encoding='UTF-8'),
                         help="Output file name.")
+    parser.add_argument("--overlap", default="error",
+                        help="What to do when files overlap (error, ignore, replace). "
+                             "See IntelHex::merge() for more info.")
```

Comment:
This is C++ style; how about `IntelHex.merge()`?

Reply:
Thanks, will do.

Reply:
fixed